dev/tools/gen_defaults/lib/drawer_template.dart

```diff
@@ -27,15 +28,19 @@ class _${blockName}DefaultsM3 extends DrawerThemeData {
   // This don't appear to be tokens for this value, but it is
   // shown in the spec.
   @override
-  ShapeBorder? get shape => const RoundedRectangleBorder(
-    borderRadius: BorderRadius.horizontal(right: Radius.circular(16.0)),
+  ShapeBorder? get shape => RoundedRectangleBorder(
+    borderRadius: const BorderRadiusDirectional.horizontal(
+      end: Radius.circular(16.0),
+    ).resolve(direction),
   );
 
   // This don't appear to be tokens for this value, but it is
   // shown in the spec.
```

Comment:
I know this wasn't part of your work here, but can we fix the wording here:

```suggestion
  // There isn't currently a token for this value, but it is shown in the spec,
  // so hard coding here for now.
```

If you don't want to include this as part of this PR, I can do it next time I am in the drawer code.